relay-server/src/actors/processor.rs

```diff
@@ -2434,11 +2439,115 @@ mod tests {
         CommonTags, TransactionMeasurementTags, TransactionMetric,
     };
     use crate::metrics_extraction::IntoMetric;
-
     use crate::testutils::{self, create_test_processor};
 
+    #[cfg(feature = "processing")]
+    use {
+        relay_metrics::BucketValue,
+        relay_quotas::{Quota, ReasonCode},
+        relay_test::mock_service,
+    };
+
     use super::*;
 
+    /// Ensures that if we ratelimit one batch of buckets in [`EncodeMetrics`] message, it won't
+    /// also ratelimit the next batches in the same message automatically.
+    #[cfg(feature = "processing")]
+    #[tokio::test]
+    async fn test_ratelimit_per_batch() {
+        let rate_limited_org = 0;
```

Comment:
IIRC we set org_id=0 to define some rate limits, so it'd be great if we don't use that ID for this test.

Reply:
ok, changed it ^^